.github/workflows/workflow.yaml

```diff
@@ -125,8 +136,19 @@ jobs:
           make e2e-build-load-image IMG=gatekeeper-e2e-helm:latest
           make e2e-helm-deploy HELM_REPO=gatekeeper-e2e-helm HELM_RELEASE=latest HELM_VERSION=${{ matrix.HELM_VERSION }}
           make test-e2e
-          echo -e '\n\n======= manager logs =======\n\n'
-          kubectl logs -n gatekeeper-system -l control-plane=controller-manager
+
+      - name: Save logs
+        run: |
+          kubectl logs -n gatekeeper-system -l control-plane=controller-manager --tail=-1 > logs-helm-${{ matrix.HELM_VERSION }}-controller.json
+          kubectl logs -n gatekeeper-system -l control-plane=audit-controller --tail=-1 > logs-helm-${{ matrix.HELM_VERSION }}-audit.json
+
+      - name: Upload artifacts
```

Comment:
Where do the artifacts get uploaded to?

Reply:
they are in the artifacts section below when you click on the example above